README.md

```diff
@@ -15,8 +15,10 @@ There's also the [PLAIN_LIST](https://github.com/LiamDawe/steamdeck_spamblock/bl
 # Contributing
 
 * To have a site accepted in a pull request, please list examples of their spamminess.
-* Please add the site to the PLAIN_LIST as well in your PR.
-* If able, please run the `./regenerate_filters.sh` script to update the LIST file with filters for the newly added site.
+* Please add the site to the `PLAIN_LIST` as well in your PR.
+* Please don't edit the `LIST` file manualy, as it is automaticaly generated.
+* If tou know what you are doing, edit `regenerate_filters.sh
```

Comment:
There’s a small spelling error here. Would you mind editing? :)

Reply:
Yes, it's resolved (I think, I'm not a native English speaker)